json-guides/circuit-breaker.json

```diff
@@ -185,7 +185,7 @@
           "title": "Configuring a delay",
           "TOCIndent": 1,
           "description": [
-              "By default, the circuit remains open for <code>5000 ms</code>. During this period, all requests to the main service are blocked and fail immediately. At the end of this period, the circuit changes to the half-open state and checks if the main service is healthy. This time period can be configured with the delay parameter.",
+              "By default, the circuit remains open for <code>5000 ms</code>. During this period, all requests to the main service are blocked and fail immediately. At the end of this period, the circuit changes to the half-open state which is a temporary state where further requests to the main microservice are allowed through and checked for success or failure. This time period can be configured with the delay parameter.",
```

Comment:
Change    "By default, the circuit remains open for <code>5000 ms</code>. During this period, all requests to the main service are blocked and fail immediately. At the end of this period, the circuit changes to the half-open state which is a temporary state where further requests to the main microservice are allowed through and checked for success or failure. This time period can be configured with the delay parameter." to    "By default, the circuit remains open for <code>5000 ms</code>. You can configure the number of milliseconds with the delay parameter. During the period that the circuit remains open, all requests to the main service are blocked and fail immediately. At the end of this period, the circuit changes to the half-open state. This half-open state is a temporary state where further requests to the main microservice are allowed through and checked for success or failure."